gateway/src/services/logger.ts

```diff
@@ -68,10 +69,23 @@ const toStdout = new winston.transports.Console({
   format: sdtoutFormat,
 });
 
+const reportingProxy = new TelemetryTransport({
+  host: 'https://api.coinalpha.com',
```

Comment:
Please don't add "https://" to the host field. Refer to the URI standard.

https://en.wikipedia.org/wiki/Uniform_Resource_Identifier